src/foam/nanos/logger/StdoutLogger.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.logger;
+
+import foam.nanos.logger.Logger;
+import java.util.logging.*;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.io.IOException;
+
+public class StdoutLogger
+  implements Logger
+{
+  protected java.util.logging.Logger logger;
+
+  private static final ThreadLocal<SimpleDateFormat> sdf = new ThreadLocal<SimpleDateFormat>() {
+    @Override
+    protected SimpleDateFormat initialValue() {
+      return new SimpleDateFormat("yyyy.MM.dd.HH.mm.ss");
+    }
+  };
+
+  private ThreadLocal<StringBuilder> sb = new ThreadLocal<StringBuilder>() {
+    @Override
+    protected StringBuilder initialValue() {
+      return new StringBuilder();
+    }
+
+    @Override
+    public StringBuilder get() {
+      StringBuilder b = super.get();
+      b.setLength(0);
+      return b;
+    }
+  };
+
+  public void start() {
```

Comment:
If I used one of these loggers directly by specifying the serviceClass I would expect them to be automatically started, like the other NanoServices. If the implemented the NanoService interface, which only has the start() method, then when instantiated directly as services, they would be automatically started. The CompositeLogger could check if each of its delegates implements NanoService and start them when it is itself started.